animations/lib/src/basics/04_tweens.dart

```diff
@@ -0,0 +1,65 @@
+import 'package:flutter/material.dart';
+
+class TweenDemo extends StatefulWidget {
+  static const String routeName = '/basics/tweens';
+
+  _TweenDemoState createState() => _TweenDemoState();
+}
+
+class _TweenDemoState extends State<TweenDemo>
+    with SingleTickerProviderStateMixin {
+  static const Duration _duration = Duration(seconds: 1);
+  static const double accountBalance = 1000000;
+  AnimationController controller;
+  Animation<double> animation;
+
+  void initState() {
+    super.initState();
+
+    controller = AnimationController(vsync: this, duration: _duration)
+      ..addListener(() {
+        // Marks the widget tree as dirty
+        setState(() {});
+      });
+    animation = Tween(begin: 0.0, end: accountBalance).animate(controller);
```

Comment:
This is a great demo from a simplicity standpoint, especially since you're writing out the value of the tween rather than just using it as a property.

I'm wondering, though, if it's missing an opportunity to show people one of Tween's most important capabilities -- mapping the controller's double to other types. Is there a way to add a ColorTween or some other interesting change of types and show that as well (two Tweens, in other words)?

Reply:
Let's do that in a separate sample

Reply:
👍